res/template/config.php.txt

```diff
@@ -11,7 +11,9 @@ $CFG->dbname    = '{{DBNAME}}';
 $CFG->dbuser    = '{{DBUSER}}';
 $CFG->dbpass    = '{{DBPASS}}';
 $CFG->prefix    = 'mdl_';
-$CFG->dboptions = [];
+$CFG->dboptions = [
+    'dbport' => '{{DBPORT}}',
```

Comment:
Hi @geichelberger and @stronk7,
should be `dbport` a string or an integer?

Moodle [uses](https://github.com/moodle/moodle/blob/d9632ca8b28304d5188a8310480f0cd6bc06d9d8/config-dist.php#L59-L61) a string, wondering if it is just for an "easy" fallback to the default port which in turns it's an actual empty string in the CLI command.

HTH,
Matteo

Reply:
I would say using string is OK here, as it does not seem important. It is responsibility of db engine class representing Moodle database interface to handle port param for connection: for MySQL it is [casted to int](https://github.com/moodle/moodle/blob/master/lib/dml/mysqli_native_moodle_database.php#L555), while for Postgres, it is [treated as string](https://github.com/moodle/moodle/blob/master/lib/dml/pgsql_native_moodle_database.php#L157).